src/main/java/frc/robot/Constants.java

```diff
@@ -104,6 +104,28 @@ public static final class ModuleConstants {
         public static final int ANGLE_CURRENT_LIMIT = 25;
     }
 
+    public static final class ClimberConstants {
+        //gains
+        public static final Gains LEFT_GAINS = new Gains(0, 0, 0);
+        public static final Gains RIGHT_GAINS = new Gains(0, 0, 0);
+
+        //movement limitation
+        public static final double kMaxLinearVelocity = 0;
+        public static final double kMaxLinearAcceleration = 0;
+
+        public static final double kGearing = 0;
+        public static final double kDriveWheelRaius = 0;
+
+        public static final double kRotToMeter = 2 * PI * kDriveWheelRaius * kGearing;
+        public static final double kRotToMeterPerSec = kRotToMeter / 60;
+        public static final int leftID = 0;
+        public static final int rightID = 0;
+        public static final double LEFT_ARM_LOCATION = 0;
+        public static final double RIGHT_ARM_LOCATION = 0;
```

Comment:
what are these values? what do they represent? if it's the encoder values then they shouldn't be final or in the constants..